website/_includes/head.html

```diff
@@ -0,0 +1,14 @@
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>{{ page.title }} | TypeDoc</title>
+    <meta name="description" content="A documentation generator for TypeScript projects." />
+    <meta name="keywords" content="TypeScript documentation generator javascript open source" />
+
```

Comment:
Perhaps add some og tags? It will help typedoc gain more exposure and be shared more easily.

```suggestion
<!--facebook-->
<meta property="og:url" content="{{ page.url | absolute_url }}">
<meta property="og:description" content="A documentation generator for TypeScript projects.">
<meta property="og:title" content="{{ page.title }}">
<meta property="og:type"   content="website" /> 
<meta property="og:site_name" content="{{ site.title }}">
<meta property="og:image"  content="{{ "/images/logo-256.png" | relative_url }}" /> 

<!--google+-->
<meta itemprop="name" content="{{ page.title }}">
<meta itemprop="description" content="A documentation generator for TypeScript projects.">
  
<!--Twitter-->
<meta name="twitter:card" content="{{ site.title }}">
<meta name="twitter:url" content="{{ page.url | absolute_url }} ">
<meta name="twitter:title" content="{{ page.title }}">
<meta name="twitter:description" content="A documentation generator for TypeScript projects.">
```